src/analyses/call_graph_helpers.cpp

```diff
@@ -0,0 +1,62 @@
+/*******************************************************************\
+
+Module: Function Call Graph Helpers
+
+Author: Chris Smowton, chris.smowton@diffblue.com
+
+\*******************************************************************/
+
+/// \file
+/// Function Call Graph Helpers
+
+#include "call_graph_helpers.h"
+
+static std::set<irep_idt> get_neighbours(
+  const call_grapht::directed_grapht &graph,
+  const irep_idt &function,
+  bool forwards)
+{
+  std::set<irep_idt> result;
+  const auto &fnode = graph[*(graph.get_node_index(function))];
+  const auto &neighbours = forwards ? fnode.out : fnode.in;
+  for(const auto &succ_edge : neighbours)
+    result.insert(graph[succ_edge.first].function);
+  return result;
+}
+
+std::set<irep_idt> get_successors(
```

Comment:
Consider calling this get_called_functions since we are specifically dealing with a call graph?